prebid-mobile/modules/rendering/android-sdk-integration-pb.md

```diff
@@ -58,30 +91,33 @@ adUnit?.fetchDemand { bidInfo ->
 }
 ```
 
-The `BidInfo` provides the following properties:
+The `bidInfo` object provides the following properties:
 
-* `resultCode` - the object of type `ResultCode` describing the status of the bid request.
+* `resultCode` - an object of type `ResultCode` describing the status of the bid request.
 * `targetingKeywords` - the targeting keywords of the winning bid
 * `exp` - the number of seconds that may elapse between the auction and the actual impression. In this case, it indicates the approximate TTL of the bid in the Prebid Cache. Note that the actual expiration time of the bid will be less than this number due to the network and operational overhead. The Prebid SDK doesn't make any adjustments to this value.
-* `nativeAdCacheId` - the local cache ID of the winning bid. Applied only to the `native` ad format.
-* `events` - the map of some publically available event URLs attached to the bid. These can be used to enable Prebid Server-based analytics when the Prebid Universal Creative (PUC) is not involved in the rendering process. If the PUC is used for rendering, it will take care of hitting these events. These are the available event URLs:
+* `nativeAdCacheId` - is the local cache ID of the winning bid. Applied only to the `native` ad format.
+* `events` - the map of publically available event URLs attached to the bid. These can be used to enable Prebid Server-based analytics when the Prebid Universal Creative (PUC) is not involved in the rendering process. If the PUC is used for rendering, it will take care of hitting these events. These are the available event URLs:
+
   * **EVENT_WIN** - this bid was chosen by the ad server as the one to display. This is the main metric for banner and native. This returns the OpenRTB `seatbid.bid.ext.prebid.events.win` field. (requires SDK v2.1.6)
   * **EVENT_IMP** - the ad creative for this bid was actually displayed. This is often the main metric for video ads. This returns the OpenRTB `seatbid.bid.ext.prebid.events.imp` field. (requires SDK v2.1.6)
 
 Code sample to extract the events:
 
-``` kotlin
+```kotlin
 val win = bidInfo.events.get(BidInfo.EVENT_WIN)
 val imp = bidInfo.get(BidInfo.EVENT_IMP)
 ```
 
-## Rendering API
+## Initializing ad auctions and rendering ads
 
-The integration and usage of the Rendering API is similar to any other Ad SDK. It sends the bid requests to the Prebid Server and renders the winning bid.
+![Ad Rendering Illustration](/assets/images/prebid-mobile/modules/rendering/Prebid-In-App-Bidding-Overview-Pure-Prebid.png)
 
-![Rendering with GAM as the Primary Ad Server](/assets/images/prebid-mobile/modules/rendering/Prebid-In-App-Bidding-Overview-Pure-Prebid.png)
+After you have made an ad request(1), your ad server will run an ad auction (2) and return the winning bid(3). You will then have to render the winning ad in your app(4).
```

Comment:
As for iOS there is no ad server and ad requests here. We use Prebid Server and make bid requests or partial bid requests.

Reply:
@charlie-docwriter - change to "After you have made an auction request(1), Prebid Server will run an ad auction (2) ..."